API_CHANGES.md

```diff
@@ -1,3 +1,24 @@
+# 10.2
+
```

Comment:
Maybe add a section `## Breaking changes` mentioning none? I would find this extremely satisfactory :)